### PR TITLE
Fixed for Raspberry Pi 3

### PR DESCRIPTION
**Pull request by SA7BNT**
Add armv71 (Raspberry Pi 3) & python-dev package 
Remove space from line 34

### CODE REVIEW

**Comment by SA7BNT** [comment]
Fixed issue arm6 to armv6

**Comment by filipsPL** [comment]
Also travis-ci says it doesn't work:

```
$ ./genpasstable.py
Traceback (most recent call last):
  File "./genpasstable.py", line 16, in <module>
    from autowx2_functions import *
  File "/home/travis/build/filipsPL/autowx2/autowx2_functions.py", line 16, in <module>
    import predict
ImportError: No module named predict
The command "./genpasstable.py" exited with 1.
```

File: install.sh [hunk 1]
```diff
@@ -31,10 +31,10 @@ echo
 sudo apt-get update
 sudo apt-get install -y rtl-sdr git libpulse-dev qt4-qmake fftw3 libc6 libfontconfig1 libx11-6 libxext6 libxft2 libusb-1.0-0-dev \
 libavahi-client-dev libavahi-common-dev libdbus-1-dev libfftw3-single3 libpulse-mainloop-glib0 librtlsdr0 librtlsdr-dev \
-libfftw3-dev  libfftw3-double3 lame sox libsox-fmt-mp3 libtool automake python-pil python-imaging imagemagick
+libfftw3-dev libfftw3-double3 lame sox libsox-fmt-mp3 libtool automake python-pil python-imaging imagemagick python-dev
 
 
-if [ ${MACHINE_TYPE} == 'armv6l' ]; then
+if [ ${MACHINE_TYPE} == 'armv61' ] || [ ${MACHINE_TYPE} == 'armv7l' ]; then
```
[the remark this answers is not in the document]
Comment (SA7BNT) [reply install.sh]: Yes my fail Sorry :(
armv61 is right

**Comment by SA7BNT** [comment]
> Also travis-ci says it doesn't work:
> 
> ```
> $ ./genpasstable.py
> Traceback (most recent call last):
>   File "./genpasstable.py", line 16, in <module>
>     from autowx2_functions import *
>   File "/home/travis/build/filipsPL/autowx2/autowx2_functions.py", line 16, in <module>
>     import predict
> ImportError: No module named predict
> The command "./genpasstable.py" exited with 1.
> ```

Can´t work with 
if [ ${MACHINE_TYPE} == 'armv61' ] || [ ${MACHINE_TYPE} == 'armv7l' ]; then

Must be
if [ ${MACHINE_TYPE} == 'armv61' ] || [ ${MACHINE_TYPE} == 'armv71' ]; then

Will fix it now

**Comment by filipsPL** [comment]
I guess this was not the case. Please look at what `uname` is saying at your Pi3 system. In my case it is:

```bash
root@raspberrypi:/usr/local/bin/autowx2# uname -m
armv7l
```
It is **armv7l** with L at the end...

**Comment by SA7BNT** [comment]
> I guess this was not the case. Please look at what `uname` is saying at your Pi3 system. In my case it is:
> 
> ```shell
> root@raspberrypi:/usr/local/bin/autowx2# uname -m
> armv7l
> ```
> 
> It is **armv7l** with L at the end...

Yes just updated

**Pull request merged**